cookbooks/aws-parallelcluster-byos/recipes/config.rb

```diff
@@ -15,6 +15,9 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+# configure system user
+include_recipe "aws-parallelcluster-byos::config_user"
```

Comment:
I think it's better to add  these users early in the process, so `init` recipes should be the right place where to call this

Reply:
good call, move to init recipe